.github/workflows/run-kata-deploy-tests-on-aks.yaml

```diff
@@ -44,6 +44,7 @@ jobs:
       GH_PR_NUMBER: ${{ inputs.pr-number }}
       KATA_HOST_OS: ${{ matrix.host_os }}
       KATA_HYPERVISOR: ${{ matrix.vmm }}
+      KUBERNETES: "vanilla"
```

Comment:
silly question: why vanilla and not aks?

Reply:
Because this will be also used for self-deployed k8s on baremetal nodes.